.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "endOfLine": "lf",
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "arrowParens": "avoid"
+}
```

Comment:
Do we need to customise prettier? I know I'm the one that tends to over-configure things, but can we leave this empty?

Reply:
FWIW I pulled this from our /learn repo - figured keeping the style matching as much as possible was a good idea.

Reply:
Okay, that's a good enough reason for me.